huaweicloud/services/eip/resource_huaweicloud_vpc_bandwidth.go

```diff
@@ -257,7 +288,16 @@ func resourceVpcBandWidthV2Update(ctx context.Context, d *schema.ResourceData, m
 		}
 	}
 
-	if d.HasChange("auto_renew") {
+	if d.HasChange("charging_mode") {
```

Comment:
if we extend size and update charging_mode to prePaid, I will add extend param in the request body in Update API.

so we should process charging_mode firstly or use the old value of `charging_mode` to update the bandwidth.